wpilib/wpilib/drive/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .differentialdrive import *
+from .killoughdrive import *
+from .mecanumdrive import *
+from .robotdrivebase import *
+from .vector2d import *
```

Comment:
Just noticed this when I tested generating the docs (for no particular reason).

Do we really need/want Vector2d accessible as `wpilib.Vector2d`?

Reply:
Maybe? If teams want access to it outside of robotdrive classes?

Reply:
I can't see a scenario where a team would want to use it over numpy…